packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -105,10 +113,27 @@ class DatePicker extends React.Component {
       selectedDate: DateUtil.createSafeDate(props.selectedDate),
     };
 
+    this.isDefaultDateAcceptable = true;
```

Comment:
Minor:

In my opinion, it's usually better to assume the data is not acceptable until you verify that it is. I think this should default to false.

Reply:
Updated: https://github.com/cerner/terra-core/pull/974/commits/f120301502e3d390e72a669fce130604de572ae5